core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -56,9 +56,15 @@ private TableProperties() {}
   public static final String PARQUET_COMPRESSION = "write.parquet.compression-codec";
   public static final String PARQUET_COMPRESSION_DEFAULT = "gzip";
 
+  public static final String PARQUET_WRITE_MODE = "write.parquet.write-mode";
+  public static final String PARQUET_WRITE_MODE_DEFAULT = "overwrite";
```

Comment:
👍 nice, having the "overwrite" default makes this change backwards compatible, right?

Reply:
For Parquet - yes, initially it was `overwrite`. For Avro - no, initially it was `create`. I made both `overwrite` to be the same but I can default Avro to `create`  for backward compatibility if needed.